### ISSUE TITLE
3rd party plugin still not working

### ISSUE THREAD

**Issue by milg0** [phonegap/phonegap-app-developer#372]
Hey guys,

I want to use the [Cordova Native Audio Plugin](https://github.com/floatinghotpot/cordova-plugin-nativeaudio) in my PhoneGap app. If I use the Android PhoneGap Developer App (1.5.6), the plugin will be undefined. But if I build the debug apk, it works great. And according to [timkim](https://github.com/phonegap/phonegap-app-developer/issues/247#issuecomment-104072807), adding the plugin should be really easy.

So, what did I do?
I installed the plugin via 'cordova plugin add cordova-plugin-nativeaudio' (also tried --save option, doesn't change anything) and try to use the plugin with
```
<script>
    document.addEventListener('deviceready', onDeviceReady);

    function onDeviceReady() {
        console.log("device ready!");
        console.log("NativeAudio is installed: "+window.plugins.NativeAudio);
        if (window.plugins && window.plugins.NativeAudio) {
            console.log("plugin ready");
            // ...
        }
    }
</script>
```
Then I start the app on my local computer with 'phonegap serve' and connect with PhoneGap Developer App but the plugin-object is just undefined. The console logs:
```
[phonegap] [console.log] device ready!
[phonegap] [console.log] NativeAudio is installed: undefined
```
So, where is the fault?

**Comment by surajpindoria** [comment]
Hi @milg0,

The reason the plugin is returning undefined is that we do not include it within the version of the Dev App we distribute. If you would like to use this plugin, add it to `config.xml` and then run the following command `npm run phonegap -- run <platform>`.

We use npm run scripts now so running regular phonegap commands will not work properly. I have just tested this and the plugin seems to be working.

**Comment by cegonzal91** [comment]
Also I have the same issue.... and how can I add it to config.xml? For example I'm trying to work with the plug screen-orientation awnd I try to Use this line to add in config.xml but continue without work with the app developer:
<plugin name="cordova-plugin-screen-orientation" version="1.3.1" />

**Comment by surajpindoria** [comment]
@cegonzal91, `cordova-plugin-screen-orientation` does not have a version 1.3.1 available on npm, therefore it is unable to find and install the plugin. Try installing using either version 1.3.6 or 1.4.0.

**Comment by surajpindoria** [comment]
I am going to close this issue, if you are still having issues feel free to reopen.

**Comment by gabriel-ir** [comment]
Hi, I am having a similar issue. I am trying to use this plug in: cordova-plugin-clipboard-with-Windows

I added it by running the following command in CLI:

 phonegap plugin add https://github.com/bytronaviation/CordovaClipboard

Then I added the XML Tag:

<plugin name="com.verso.cordova.clipboard" source="npm" spec="~1.0.3" />	

And also added the script tag in the index.html:

<script type="text/javascript" src="clipboard.js"></script>

But it doesn't work. Here is my code:

var text = "Hello World!";
cordova.plugins.clipboard.copy(text);

I simply does nothing.

If I di this:
alert(cordova.plugins);
alert(cordova.plugins.clipboard);

"cordova.plugins" is actually an object, but "cordova.plugins.clipboard" is undefined.

What am I doing wrong?
